styles.css

```diff
@@ -66,7 +66,16 @@
 
 .grvsc-line-highlighted::before {
   background-color: var(--grvsc-line-highlighted-background-color, transparent);
-  box-shadow: inset var(--grvsc-line-highlighted-border-width, 4px) 0 0 0 var(--grvsc-line-highlighted-border-color, transparent);
+  box-shadow: inset var(--grvsc-line-highlighted-border-width, 4px) 0 0 0
+    var(--grvsc-line-highlighted-border-color, transparent);
+}
+
+.grvsc-line-diff-add {
+  background-color: var(--grvsc-line-diff-add-background-color, transparent);
```

Comment:
Since we’re putting this in v3, go ahead and pick default colors that you think will work well 🎉 

With the layout changes I made, though, I think you’ll need to use a pseudoelement like I changed line highlighting to use. Reason is that the `.grvsc-line` span is no longer guaranteed to be as wide as the container, unfortunately, so this will result in the highlight stopping at the end of the text instead of extending all the way to the right edge of the container.

I think if a line is both highlighted and diff-highlighted, the diff highlight should have priority. Do you think that makes sense? (I expect this is a severe edge case, but probably something that should be tested/handled nonetheless.)

Reply:
Re pseudoelement: you can reuse the existing `.grvsc-has-line-highlighting` container class to generate the `::before` element, then set its background-color on `.grvsc-line-diff-add::before`.

Reply:
> Since we’re putting this in v3, go ahead and pick default colors that you think will work well 🎉

I went with `rgba(0, 255, 60, 0.4)` (additions) and `rgba(255, 0, 20, 0.4)` (deletions) for now. They are similar (but not identical) to the colors used on GitHub.

> I think if a line is both highlighted and diff-highlighted, the diff highlight should have priority. Do you think that makes sense?

Sounds sensible.

> Re pseudoelement: you can reuse the existing `.grvsc-has-line-highlighting` container class to generate the `::before` element, then set its background-color on `.grvsc-line-diff-add::before`.

I tried. Not sure I did this right. Let me know if something's off.

Reply:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/3277153/82272822-86f67480-9930-11ea-9f01-201ce9599812.png">

Looks right!